test/main.test.js

```diff
@@ -4,6 +4,7 @@ const gotScraping = require('../src');
 
 const { startDummyServer } = require('./helpers/dummy-server');
 
+process.env.APIFY_PROXY_PASSWORD = 'rgC8JJ8NcrDnDdwsxDqWz7jKS';
```

Comment:
Is this a real proxy password? If it is, please invalidate it.

Reply:
Oh, done. 🤦